data/data/bootstrap/bootstrap-in-place/files/usr/local/bin/bootstrap-in-place.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -euoE pipefail ## -E option will cause functions to inherit trap
+
+# This script is executed by bootkube.sh when installing single node with bootstrap in place
+CLUSTER_BOOTSTRAP_IMAGE=$1
+
+
+bootkube_podman_run() {
+  # we run all commands in the host-network to prevent IP conflicts with
+  # end-user infrastructure.
+  podman run --quiet --net=host "${@}"
+}
+
+if [ ! -f stop-etcd.done ]; then
+  echo "Stop etcd static pod by moving the manifest"
+  mv /etc/kubernetes/manifests/etcd-member-pod.yaml /etc/kubernetes
+
+  until ! crictl ps | grep etcd; do
+    echo "Waiting for etcd to go down"
+    sleep 10
+  done
+
+  touch stop-etcd.done
+fi
+
+if [ ! -f master-ignition.done ]; then
+  echo "Creating master ignition and writing it to disk"
+  # Get the master ignition from MCS
+  curl --header "Accept:'application/vnd.coreos.ignition+json;version=3.1.0 ;q=0.1'" \
```

Comment:
No need for the `;q=0.1`, is there?  [RFC 7231][1] has the weight coming in via an optional `accept-params`.  I also don't see the RFC asking for your `'` quoting.  So I'd expect:

```sh
curl --header 'Accept:application/vnd.coreos.ignition+json;version=3.1.0' \
  http://localhost:22624/config/master -o /opt/openshift/original-master.ign
```

Am I just missing something?

[1]: https://tools.ietf.org/html/rfc7231#section-5.3.2

Reply:
You are right.